src/loading-scene.ts

```diff
@@ -1,11 +1,15 @@
+// -- start tsdoc imports --
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { type UiWindowType } from "#enums/ui-window-type";
+// -- end tsdoc imports --
```

Comment:
NIT: for convenience to the next dev use multi-line enable/disable


```suggestion
// -- start tsdoc imports --
/* eslint-disable @typescript-eslint/no-unused-vars */
import { type UiWindowType } from "#enums/ui-window-type";
/* eslint-enable @typescript-eslint/no-unused-vars */
// -- end tsdoc imports --
```